randovania/games/samus_returns/logic_database/Area 8.txt

```diff
@@ -1187,9 +1187,15 @@ Extra - asset_id: collision_camera_020
   * Layers: default
   * Open Passage to Queen Arena Access/Dock to Queen Arena
   > Before Queen
-      After Area 8 - Larva Metroid 1 and After Area 8 - Larva Metroid 10 and After Area 8 - Larva Metroid 2 and After Area 8 - Larva Metroid 3 and After Area 8 - Larva Metroid 4 and After Area 8 - Larva Metroid 5 and After Area 8 - Larva Metroid 6 and After Area 8 - Larva Metroid 7 and After Area 8 - Larva Metroid 8 and After Area 8 - Larva Metroid 9
-  > Dock to Hatchling Room
-      After Area 8 - Queen Metroid or Enabled Remove Queen Wall
+      All of the following:
+          After Area 8 - Larva Metroid 1 and After Area 8 - Larva Metroid 10 and After Area 8 - Larva Metroid 2 and After Area 8 - Larva Metroid 3 and After Area 8 - Larva Metroid 4 and After Area 8 - Larva Metroid 5 and After Area 8 - Larva Metroid 6 and After Area 8 - Larva Metroid 7 and After Area 8 - Larva Metroid 8 and After Area 8 - Larva Metroid 9
```

Comment:
template?

Reply:
I can see why you'd use one, but it's only used one time, so eh.

Reply:
makes things easier if you need to move this again.